### PR TITLE
Add Shopping list application with nextjs

### PR DESCRIPTION
**Pull request by rwxsb**
An aspire sample shopping list application done with Next.js

### CODE REVIEW

**Comment by rwxsb** [comment]
> @rwxsb please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @dotnet-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @dotnet-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @dotnet-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@dotnet-policy-service agree

File: samples/Listly/Listly.AppHost/NodeAppHostingExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+using Aspire.Hosting.Lifecycle;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Hosting;
+
+namespace Listly.AppHost;
+
+internal static class NodeAppHostingExtension
```
Comment (davidfowl) [inline samples/Listly/Listly.AppHost/NodeAppHostingExtensions.cs:8]: This is built in now so you can delete it

File: samples/Listly/Listly.AppHost/Program.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+using Listly.AppHost;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata.Internal;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var listlydb = builder.AddPostgresContainer("listly", 1433, "1q2w3e4r5T*")
```
Comment (davidfowl) [inline samples/Listly/Listly.AppHost/Program.cs:6]: Why are the port and password specified  here?

File: samples/Listly/Listly.Database/Listly.Database.csproj [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="8.0.0">
+          <PrivateAssets>all</PrivateAssets>
+          <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+        <PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0"/>
+    </ItemGroup>
+
+    <ItemGroup>
+      <Reference Include="Microsoft.EntityFrameworkCore">
+        <HintPath>..\..\..\.nuget\packages\microsoft.entityframeworkcore\8.0.0\lib\net8.0\Microsoft.EntityFrameworkCore.dll</HintPath>
```
Comment (davidfowl) [inline samples/Listly/Listly.Database/Listly.Database.csproj:21]: Why is there a hint path?

**Comment by DamianEdwards** [comment]
I don't think this sample adds enough that's meaningfully different from the existing Node.js and JavaScript app samples anymore so closing it out. Adding a Next.js app to the existing JavaScript apps samples might be interesting though so feel free to log an issue for that for further discussion.

**Pull request closed without merging**